egs/tedlium/asr1/run.sh

```diff
@@ -3,13 +3,18 @@
 # Copyright 2017 Johns Hopkins University (Shinji Watanabe)
 #  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
 
+#Multi-GPU configuration
+export num_gpu = 1 # number of gpu cores to be used
+gpu=-1         # use 0 when using GPU on slurm/grid engine, use "[0, 1]" when you want to use 2 cores, otherwise -1
```

Comment:
Is it possible to integrate `num_gpu` and `gpu` ?

Reply:
I agree. Maybe using only num_gpu is the simplest solution. In addition, I prefer num_gpu=0 for CPU computation and use `model.cuda(device_ids=range(num_gpu))` because the environment variable `CUDA_VISIBLE_DEVICES` can handle [the device id mappings](http://acceleware.com/blog/cudavisibledevices-masking-gpus) and some job tools (slurm, torque, or JHU /usr/local/bin/free-gpu) specify CUDA_VISIBLE_DEVICES automatically. 

I propose to use
- num_gpu for specifying the number of GPUs (0 for CPU)
- CUDA_VISIBLE_DEVICES defined at the outside of run.sh for specifying the GPU ids